src/layout/app.css

```diff
@@ -1,26 +1,23 @@
 
-html {
+html, body {
   background-color: #f3f3f3;
   font-family: Roboto, Arial, sans-serif;
+  padding: 0;
+  margin: 0;
 }
```

Comment:
I guess stylistically I'd prefer to not repeat the bg color and font-family here. But it's minor.